docs/embedded.md

```diff
@@ -0,0 +1,87 @@
+# Embedded C#/WinRT Support
+
+## Overview
+
+Embedded support is a C#/WinRT feature that allows you to have the WinRT layer of your projection compiled into the projection itself. This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. 
+
+Enabling embedded support brings in the WinRT.Runtime sources to the project and automatically sets them to be compiled with the project.
+Any self-contained usage of Windows SDK types must be specified using the property `CsWinRTIncludes`. The specified types will be projected and embedded into the project's *.dll*. This means projects using embedded support no longer have a dependency on `Microsoft.Windows.SDK.NET.dll` and `WinRT.Runtime.dll`.
+
+For an example, you can look at the [sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/TestEmbedded). 
+
+## Scenarios
+

```

Comment:
Maybe list out the reasons/ benefits of using embedded support here? E.g. 

Reasons for using C#/WinRT embedded support include:

- Decreased component/app size: The developer only needs to include the minimum necessary parts of the Windows SDK required by their component or app, reducing the output size significantly.
- Downlevel support: App developers on .NET 5+ can target Windows 7 (`net5.0-windows`) and light-up on Windows 10/11 when consuming an embedded .NET 5+ library. This also removes the need for multi-targeting (e.g. `net5.0-windows` and `net5.0-windows10.xxxxx.0`). 
- Remove the need for multi-targeting: Library authors can support all .NET Standard 2.0 app consumers (which includes .NET 5+) without the need for multi-targeting. App consumers are able to target any .NET Standard 2.0 compatible TFM (e.g., `netcoreapp3.1`, `net48`, `net5.0-windows`)